### DISCUSSION TITLE
Starstruck badge not showing all repos.

### DISCUSSION THREAD

**Discussion by Beercow** [community/community#35464] [Other Feature Feedback, Questions, & Ideas]
My Starstruck badge is only showing one repository. I have at least two more with more stars than the one it is showing.

**Comment by radar-blip** [comment]
Its only supposed to show one repository, not all of them.

**Reply by Beercow** [reply]
I don’t think that’s correct. 
![E16C9224-9336-458D-B95A-D95DBAF298C8](https://user-images.githubusercontent.com/10360919/194622433-229a6bf6-874e-4927-bd83-069a94d1d0b9.jpeg)
![FA43B9F1-5C52-445D-A440-34457545D022](https://user-images.githubusercontent.com/10360919/194622439-48566011-4a4b-4724-aefa-8a4aa7580de7.jpeg)

As an example, these should also be listed but they are not. 
![732A557F-4BBC-4319-B0F7-1F240FBF4CF8](https://user-images.githubusercontent.com/10360919/194622739-86937fb5-c228-475e-b6ac-f4aafc23ccc6.jpeg)
![2E5F73FB-7942-464C-A1CF-C803EB0A784A](https://user-images.githubusercontent.com/10360919/194622742-669285ca-e8ad-4d51-80d7-43dd2908c0d0.jpeg)

**Reply by radar-blip** [reply]
Its somewhat correct. For the default Starstruck achievement you need a repo with 16 stars.
You will get the Bronze starstruck (x2) when you reach 128 stars, This could be on a different repo.

512 stars for Silver (x3)
4096 stars for Gold (x4)

Here is more info if you need it: [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Comment by jfullstackdev** [comment]
you are still not reaching the next threshold, isn't it ? 

do you have any other repo that has 128 stars ?

**Comment by Beercow** [comment]
Figured out how it works. There are four levels to the badge. The first repo to reach that level gets the slot.

**Reply by jfullstackdev** [reply]
yup, that's how it works like my comment on top, I'm calling it threshold before you receive the next achievement

**Comment by aimore** [comment]
check how it works in here
https://github.com/Schweinepriester/github-profile-achievements

**Comment by MonitSharma** [comment]
I have two repositories with over 16 stars, I don't even have a single star struck badge. Why is it so?
